P6-Gameplay-Part-2/content.md

```diff
@@ -76,10 +76,8 @@ A great place to add a steady decline would be in the `update(..)` method.
 override func update(currentTime: CFTimeInterval) {
   /* Called before each frame is rendered */
   if state != .Playing { return }
-
```

Comment:
I took another look at this, ah yes there is an annoying issue when it comes to empty lines.  The agreed 'fix' is to use a '>' on any empty, then it will render correctly.  If you can update your pull to reflect this I'll merge it.  Cheers